### ISSUE TITLE
Accessibility: Reader Search

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#10732]
###### Talkback Audit 

* Search 
	* When the search has begun the app doesn't announce this. Also when the search has ended it doesn't announce this either. The app should express the state of the search results.

**Comment by jd-alexander** [comment]
Closing this as it will be resolved in #10741
